### PR TITLE
refactor(parsing): separate parsing code from IO code. Bootstrapped tests.

### PR DESCRIPTION
**Pull request by antoine-richard**

### CODE REVIEW

**Comment by antoine-richard** [comment]
diff is hard to read. sorry...

File: parser/test/parse.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+var parse = require('../parse');
+var chai = require('chai');
+chai.use(require('chai-string'));
+var expect = chai.expect;
+
+describe('parse', function() {
+
+  it('should output CSV headers', function() {
+    let result = parse("6:10 Kill: 4 1 30: JonathanMcCow killed Cyp' by UT_MOD_AK103").value();
+    console.log(result)
```
Comment (PierreReliquet) [inline parser/test/parse.js:11]: not sure about the usefulness of the `console.log`

Comment (PierreReliquet) [reply parser/test/parse.js:11]: any reason to add it?

**Review by PierreReliquet — APPROVED** [review]
Good job

File: parser/test/parse.js (hunk 1 above)
Comment (antoine-richard) [reply parser/test/parse.js:11]: no reason at all! ;)

**Pull request merged**